### DISCUSSION TITLE
Doesnt show raw data

### DISCUSSION THREAD

**Discussion by twinko** [NightscoutFoundation/xDrip#2290] [Q&A]
Hi i enabled raw data and set it to always, but it doesnt show up. Did i missed something? I can see the raw data on my phone.

attached a screenshot_

![131-4-→](https://user-images.githubusercontent.com/11973158/186279485-34297181-1df0-42ad-bf02-f67756d11d43.png)

Edit: these are my heroku (mongodb) Config Vars

![xdrip-twinko-·-Settings-Heroku](https://user-images.githubusercontent.com/11973158/186281527-f32dc123-c4b7-46b0-bf65-e6533b64ca80.png)

**Comment by Navid200** [comment]
Isn't that a screenshot of Nightscout?
What do you see in xDrip?
And, please mention what the hardware data source is.
Thanks

**Reply by twinko** [reply]
Hi yes this is a nightscout screenshot,
attached ascreenshot from my phone running xdrip.

source is patched libre 2
![WhatsApp Image 2022-08-24 at 00 56 05](https://user-images.githubusercontent.com/11973158/186280320-23bd9d79-a007-47d8-811a-2c8ddd3094ac.jpeg)

**Reply by Navid200** [reply]
So, the raw data is shown in xDrip, right?

Is your question then:  Why the raw data is not uploaded to Nightscout?

If that's the question, I can then find the right person.
The title you chose includes the word "show".  This is the xDrip repository.  And xDrip is perfectly showing the raw data.
It helps if you help us understand your question.

**Reply by twinko** [reply]
sorry wrong repo, my bad. thanks for helping anyways :)

**Comment by Navid200** [comment]
@psonnera  Is Libre raw data uploaded to Nightscout?

**Reply by psonnera** [reply]
I don't think so.

**Reply by twinko** [reply]
The following issue sounds like the that syncs
https://github.com/NightscoutFoundation/xDrip/issues/1597
